include/tarxx.h

```diff
@@ -217,6 +285,14 @@ namespace tarxx {
             block_t zeroes {};
             write(zeroes);
             write(zeroes);
+
+            if (compression_ == compression_mode::lz4 && lz4_ctx_ != nullptr) {
+                const auto lz4_result = lz4_check_error(LZ4F_compressEnd,
+                                                        lz4_ctx_->get(), lz4_out_buf_.data(), lz4_out_buf_.capacity(), nullptr);
+
+                lz4_out_buf_pos_ += lz4_result;
+                write_lz4_data();
+            }
```

Comment:
please only compile in if lz4 support is activated (see comments in `CMakeLists`)

Reply:
done :)